### PR TITLE
feat: orgs create dynamic events from org member list

### LINKED ISSUE #14519: [CAL-3399] Orgs: create dynamic events from org member list
CleanShot 2024-04-11 at 10 58 32@2x

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-3399](https://linear.app/calcom/issue/CAL-3399/orgs-create-dynamic-events-from-org-member-list)</sub>

### PR DESCRIPTION
**Pull request by sean-brydon**
Fixes #14519 
Fixes CAL-3399

CleanShot 2024-05-07 at 10 41 40
CleanShot 2024-05-07 at 10 41 43

### CODE REVIEW

**Review by Udit-takkar — COMMENTED** [review]
NIT: displays three avatar on two people dynamic links

https://github.com/calcom/cal.com/assets/53316345/82b5f493-f495-4b3c-a792-1c1e848fcdce

File: packages/features/users/components/UserTable/UserListTable.tsx [hunk 1]
```diff
@@ -335,6 +348,50 @@ export function UserListTable() {
...
           },
         ]}
+        renderAboveSelection={(table: Table<User>) => {
+          const numberOfSelectedRows = table.getSelectedRowModel().rows.length;
+          const isVisible = numberOfSelectedRows >= 2 && dynamicLinkVisible;
+
+          const users = table
+            .getSelectedRowModel()
+            .flatRows.map((row) => row.original.username)
+            .filter((u) => u !== null);
```
Comment (Udit-takkar) [inline packages/features/users/components/UserTable/UserListTable.tsx:358]: NIT:
```suggestion
            .filter(Boolean);
```

**Comment by sean-brydon** [comment]
> NIT: displays three avatar on two people dynamic links
> 
>  Screen.Recording.2024-05-07.at.3.58.08.PM.mov

I dont think this is related to this PR - all i've done here is create a way to open this booker. Correct me if im mistaken tho

File: packages/features/users/components/UserTable/UserListTable.tsx (hunk 1 above)
Comment (sean-brydon) [reply packages/features/users/components/UserTable/UserListTable.tsx]: Good spot

File: packages/ui/components/data-table/DataTableSelectionBar.tsx [hunk 2]
```diff
@@ -11,42 +12,56 @@ export type ActionItem<TData> =
...
-                </Button>
-              ) : action.type === "render" ? (
-                action.render(table)
-              ) : null}
-            </Fragment>
-          ))}
-        </motion.div>
+        <div className="fade-in fixed bottom-6 left-1/2 hidden -translate-x-1/2 gap-1 md:flex md:flex-col lg:max-w-[40%]">
+          {renderAboveSelection && renderAboveSelection(table)}
+          <div className="bg-brand-default text-brand item-center hidden justify-between rounded-lg p-2 lg:flex">
```
[the remark this answers is not in the document]
Comment (sean-brydon) [reply packages/ui/components/data-table/DataTableSelectionBar.tsx:48]: Why doesnt it just grow ffs ... Fixing <3

**Review by Udit-takkar — COMMENTED** [review]
LGTM. I'll merge after this https://github.com/calcom/cal.com/pull/14912/files#r1592218443 is fixed

**Review by Udit-takkar — APPROVED** [review]
LGTM

**Pull request merged**